Ignore prepublish output

vweevers opened the pull request:
First commit corrects the callback signature from `exec(cmd, cb(err, stderr, stdout))` to `exec(cmd, cb(err, stdout, stderr))`.

Second commit takes the tar filename from the last line of `npm pack` output. Because if you have a `prepublish` script, `npm pack` output can contain tests etc:

```
> my-package@1.0.0 prepublish /somewhere/my-package
> npm t

> my-package@1.0.0 test /somewhere/my-package
> tape test/*.js

TAP version 13
# basic
ok 1 should be equal

1..1
# tests 1
# pass  1

# ok

my-package-1.0.0.tgz
```

vweevers committed:
account for prepublish output

thlorenz commented on index.js:
```diff
@@ -53,7 +53,7 @@ function listFiles(root, out) {
 
     // npm logs created filename on stdout
-    var tarFile = path.join(process.cwd(), stdout.trim());
+    var tarFile = path.join(process.cwd(), stdout.trim().split(/\s+/).pop());
```
Wondering if it would be better to split on `'\n'` since the file will be on it's own line.
Also that would cover cases in which the filename has a white space (in case that ever happens).

thlorenz commented:
Hey thanks for catching my stdout/stderr mistake.

I made you collaborator.

So once we figure out the little nit I pointed out in the code please squash the commits and merge to master following [this guide](https://gist.github.com/thlorenz/f7df0daf13c6f6c1fa84).
I will then version and publish to npm.

Thanks.

vweevers commented:
Addressed the nit and added a test to be sure.

By the way, the test script (`tap test/*.js`) doesn't work for me, it times out. Looks like the same happened on the latest [travis build](https://travis-ci.org/thlorenz/irish-pub/jobs/58937880). I don't know the cause but a quick fix is to change the script to `node test`.

elldritch commented:
Any update on this? Would love to get this patch published, otherwise `irish-pub` breaks on basically all of my (CoffeeScript) packages.

vweevers commented:
@thlorenz this is ready to be published. The timeout issue is addressed in PR #8.
